doc/vector-extra/riscv-crypto-vector-extra-introduction.adoc

```diff
@@ -0,0 +1,10 @@
+[[crypto_vector_introduction]]
+== Introduction
+
+This document describes the proposed _vector_ _extra_ cryptography
+extensions for RISC-V.
+Those extensions extend the _vector_ cryptography extensions for RISC-V,
+providing extra features not mandatory for a high performace implementation but which
+can help further improve the efficiency of the algorithms that use them.
```

Comment:
it was suppose to be "extra features" so "new extensions".

Reply:
Well, without the extensions there are no algorithms that use them, so that phrasing is a  bit strange.  Maybe list some of the specific algorithms that are intended to be improved?